github/issues_labels.go

```diff
@@ -12,13 +12,14 @@ import (
 
 // Label represents a GitHub label on an Issue
 type Label struct {
+	ID    *int    `json:"id,omitempty"`
 	URL   *string `json:"url,omitempty"`
 	Name  *string `json:"name,omitempty"`
 	Color *string `json:"color,omitempty"`
 }
 
 func (l Label) String() string {
-	return fmt.Sprint(*l.Name)
+	return *l.Name
```

Comment:
This change is an improvement, but really, this was meant to be `return Stringify(l)`. /cc @willnorris

```Go
func (l Label) String() string {
	return Stringify(l)
}
```

If you [look at the implementations](https://cloud.githubusercontent.com/assets/1924134/24018043/b70540d4-0a68-11e7-88bd-c901e4f5757e.png) of all other `String() string` methods on the go-github types, that's what they do. This is the only unintended exception (other than `Timestamp.String`, but that's a special case on purpose).

> what was this code trying to do?

The intent is to implement `fmt.Stringer` interface and print a more readable value for debugging purposes (since all the fields are pointers). There's no other contract it needs to satisfy.